Kinvey/Kinvey/DataStore.swift

```diff
@@ -317,7 +317,7 @@ open class DataStore<T: Persistable> where T: NSObject {
             query,
             deltaSet: deltaSet,
             readPolicy: readPolicy
-        ) { (result: Result<[T], Swift.Error>) in
+        ) { (result: Result<AnyRandomAccessCollection<T>, Swift.Error>) in
```

Comment:
yes, it is... a way to avoid this breaking change is use the same pattern that Apple use, which is add a property called `lazy`, so `dataStore.lazy.find()` would have those new methods.